app/models/partner.rb

```diff
@@ -104,8 +104,8 @@ class Partner < ApplicationRecord
 
   delegate :email, to: :user
 
-  def export_json
-    {
+  def export_json(include_family_and_children_data = false)
```

Comment:
```suggestion
  def export_json(include_family_and_children_data: false)
```
I find using keyword arguments to be very useful in conveying knowledge on how to use a specific method. Using a keyword argument a call to this method can go from:
```
Partner.export_json(true)
```
to
```
Partner.export_json(include_family_and_children_data: true)
```

The 2nd provides a bit more information on what that `true` is doing.